Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -683,9 +678,10 @@ static LayoutRect sizingBox(RenderObject& renderer)
     return box.style().boxSizing() == BoxSizing::BorderBox ? box.borderBoxRect() : box.computedCSSContentBoxRect();
 }
 
-static Ref<CSSFunctionValue> matrixTransformValue(const TransformationMatrix& transform, const RenderStyle& style)
+static Ref<CSSFunctionValue> matrixTransformValue(TransformationMatrix transform, const RenderStyle& style)
```

Comment:
Why can't we keep the transform `const`? I'm not seeing where we're calling non-const methods on it.

Reply:
It's a leftover of the previous patch. Will fix. Thank you.